### PR TITLE
Some quality-of-life type of changes

### PR DESCRIPTION
**Pull request by deiwin**
Info in commits.

### CODE REVIEW

**Comment by deiwin** [comment]
Deploying without reviews to test the retry with input mechanism.

**Comment by deiwin** [comment]
!deploy

**Comment by sm-deployer** [comment]
@deiwin, your input is required [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/2/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/2//console)).

**Comment by sm-deployer** [comment]
Deploy failed or was aborted. @deiwin, please check the logs [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/2/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/2//console)).

**Comment by deiwin** [comment]
!deploy

**Comment by sm-deployer** [comment]
@deiwin, your input is required [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/4/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/4//console)).

**Comment by sm-deployer** [comment]
@deiwin, the changes were validated in acceptance. Please click **Proceed** [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/4/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/4//console)) to continue the deployment.

**Comment by sm-deployer** [comment]
@deiwin, your input is required [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/4/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/4//console)).

**Comment by sm-deployer** [comment]
@deiwin, your input is required [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/4/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/4//console)).

**Comment by sm-deployer** [comment]
Deploy failed or was aborted. @deiwin, please check the logs [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/4/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/4//console)).

**Comment by deiwin** [comment]
Aborting seemed to work well. Starting again to try getting a review after merging has already failed once.

**Comment by deiwin** [comment]
!deploy

**Comment by sm-deployer** [comment]
@deiwin, your input is required [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/5/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/5//console)).

**Comment by sm-deployer** [comment]
@deiwin, the changes were validated in acceptance. Please click **Proceed** [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/5/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/5//console)) to continue the deployment.

**Comment by sm-deployer** [comment]
@deiwin, your input is required [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/5/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-96/5//console)).

**Comment by deiwin** [comment]
Failed with the following as expected and is now waiting for input.

```
+ git push origin @:master
remote: error: GH006: Protected branch update failed for refs/heads/master.        
remote: error: At least 1 approving review is required by reviewers with write access.        
To github.com:salemove/pipeline-lib.git
 ! [remote rejected] @ -> master (protected branch hook declined)
error: failed to push some refs to 'git@github.com:salemove/pipeline-lib.git'
```

Requesting for review and proceeding after the review to test the flow that it was added for.

File: config/kubernetes/services/deploy-pipeline-test.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ spec:
   ports:
   - port: 80
     protocol: TCP
+    name: tcp
```
Comment (kuzaxak) [inline config/kubernetes/services/deploy-pipeline-test.yml]: Why not `http`?

Comment (deiwin) [reply config/kubernetes/services/deploy-pipeline-test.yml]: The `protocol` was TCP so I stuck to that. That's also the default protocol, according to [docs](https://kubernetes.io/docs/concepts/services-networking/service/#protocol-support).

Comment (kuzaxak) [reply config/kubernetes/services/deploy-pipeline-test.yml]: Yes, but the name of port != protocol.

Comment (deiwin) [reply config/kubernetes/services/deploy-pipeline-test.yml]: I don't know if kube-proxy differentiates this in either case.

Comment (kuzaxak) [reply config/kubernetes/services/deploy-pipeline-test.yml]: name used by istio to analyze the traffic, and it this case its http traffic via tcp protocol

Comment (deiwin) [reply config/kubernetes/services/deploy-pipeline-test.yml]: It works as-is so I won't start the deploy from scratch over this. Plus I wouldn't be able to test the deploy -> get review -> finish deploy flow anymore without dismissing @ostankin 's review. Can consider changing separately, but this service is only accessed during testing of this library, so it doesn't much matter what Istio thinks of its traffic.

**Pull request merged**